Package.swift

```diff
@@ -3,6 +3,9 @@ import PackageDescription
 
 let package = Package(
     name: "Weaver",
+    platforms: [
+       .macOS(.v10_13)
```

Comment:
Is this intentional?

If yes, I'm concerned that this is limiting Weaver to be used on that specific version only.

Reply:
Yes, it is. This line just moved deployment target from v10_10 (implicit) to v10_13 and resulted binary works great on the latest os version.
https://developer.apple.com/documentation/swift_packages/supportedplatform

10.13 is needed for regexp named groups:
```
NSTextCheckingResult
- (NSRange)rangeWithName:(NSString *)name API_AVAILABLE(macosx(10.13), ios(11.0), watchos(4.0), tvos(11.0));
```